console_conf/ui/views/welcome.py

```diff
@@ -29,16 +29,28 @@
 
 class WelcomeView(BaseView):
     title = "Ubuntu Core"
-    excerpt = ("Configure the network and setup an administrator "
-               "account on this all-snap Ubuntu Core system.")
+    excerpt = (
```

Comment:
I presume this is from black?

Reply:
Yeah, but the repo isn't using black, so I stopped fighting/using it eventually.

Reply:
maybe it should be black.